flyway/postgre/migrations/sql/V1_3_Create_lesson2_tables.sql

```diff
@@ -0,0 +1,85 @@
+-- Database: admindb
+
+-- DROP DATABASE IF EXISTS admindb;
+
+CREATE TYPE sex AS ENUM ('female', 'male');
+CREATE TYPE status AS ENUM ('active', 'deactived', 'deleted');
+
+DROP TABLE IF EXISTS sso_organization;
+CREATE TABLE sso_organization (
+	organization_id int GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+	organization_name TEXT,
+	kvp json,
+	date_created TIMESTAMP,
+	date_modified TIMESTAMP,
+	status status DEFAULT 'active' NOT NULL,
+	address TEXT,
+	contact_number TEXT,
+	email TEXT,
+	country TEXT,
+	city TEXT,
+	postal_code TEXT,
+	organization_sub_name TEXT,
+	website_url TEXT
+);
+
+DROP TABLE IF EXISTS sso_role;
+CREATE TABLE  sso_role(
+	role_id uuid DEFAULT gen_random_uuid() PRIMARY KEY,
```

Comment:
Nếu dùng UUID muốn gen random uuid cần cài thêm extension `CREATE EXTENSION IF NOT EXISTS "uuid-ossp";`
```suggestion
	role_id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
```